src/main/webapp/app/shared/category-selector/category-selector.component.ts

```diff
@@ -44,7 +44,8 @@ export class CategorySelectorComponent implements OnChanges {
      */
     openColorSelector(event: MouseEvent, tagItem: ExerciseCategory) {
         this.selectedCategory = tagItem;
-        this.colorSelector.openColorSelector(event);
+        const height = 110;
```

Comment:
Maybe place this value up higher in the component below the Input/Output props as a `readonly` property (e.g. as `colorSelectorHeight`) since it's more of a configuration value than a local variable of the method?

Reply:
Done in b388424253bcf5adf8d9abdbacf18b256655cd24